src/test/resources/reference-test/ChangeScanner.java

```diff
@@ -0,0 +1,39 @@
+package net.sf.jabref.collab;
+
+import java.io.*;
+import java.util.*;
+import javax.swing.*;
+import javax.swing.tree.DefaultMutableTreeNode;
+import net.sf.jabref.*;
+import net.sf.jabref.groups.*;
+import net.sf.jabref.imports.*;
+
+
+public class ChangeScanner extends Thread {
+    
+    
+    /**
+     * Finds the entry in neu best fitting the specified entry in old. If no entries get a score
```

Comment:
This is a stripped version of a single Java file in JabRef :). I think `neu` has been chosen to avoid new, which is a keyword.